tskv/src/wal/mod.rs

```diff
@@ -473,14 +500,53 @@ impl VnodeWal {
                     .delete(tenant, database, table, *vnode_id, series_ids, time_ranges)
                     .await
             }
+            writer::Task::ClearWalEntry(ClearWalEntry {
+                tenant_database,
+                sequence,
+                vnode_id,
+                ..
+            }) => {
+                let mut delete_ids = vec![];
+                let wal_files = file_manager::list_file_names(self.wal_dir());
```

Comment:
The map of wal_file_id and it's footer data `(min_seq, max_seq)` could be stored in wal manager when it rolls a wal writer, just as it did in the past versions, then we don't need to open all wal files to get them.

Reply:
We can optimize next.